rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -390,25 +396,32 @@ pub async fn install_all_nns_canisters(
 /// Arguments
 /// 1. `with_mainnet_nns_canister_versions` is a flag indicating whether the mainnet
 ///    (or, therwise, tip-of-this-branch) WASM versions should be installed.
-/// 2. `initial_balances` is a `Vec` of `(test_user_icp_ledger_account,
+/// 2. `with_test_nns_governance_canister` is a flag indicating whether the test version of
+///    the governance canister should be installed. Mutually exclusive with `with_mainnet_nns_canister_versions`.
+/// 3. `initial_balances` is a `Vec` of `(test_user_icp_ledger_account,
 ///    test_user_icp_ledger_initial_balance)` pairs, representing some initial ICP balances.
-/// 3. `custom_initial_registry_mutations` are custom mutations for the inital Registry. These
-///    mutations should comply with Registry invariants, otherwise this function will fail.
-/// 4. `neurons_fund_hotkeys` - hotkeys of the 1st NNS (Neurons' Fund-participating) neuron.
-/// 5. `install_index_canister` - a flag indicating whether the Index canister for ICP Ledger should be installed.
-/// 6. `install_cmc` - a flag indicating whether the Cycles minting canister should be installed.
+/// 4. `custom_initial_registry_mutations` are custom mutations for the inital Registry. These
+///    should mutations should comply with Registry invariants, otherwise this function will fail.
```

Comment:
it seems that "should" is redundant

Reply:
FWIW, this typo was there before my changes, but I'll fix it